### PR TITLE
Package nodejs-react-onclickoutside 6.6.3

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: nodejs-react-onclickoutside/nodejs-react-onclickoutside.spec [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+%define npm_cache_dir /tmp/npm_cache_%{name}-%{version}-%{release}
+%description
+%{summary}
+
+%prep
+%setup -q -n package
+
+%build
+$BUILD
```
Comment (ehelms) [inline nodejs-react-onclickoutside/nodejs-react-onclickoutside.spec]: I think something improper from npm2rpm

File: nodejs-react-onclickoutside/nodejs-react-onclickoutside.spec [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+%nodejs_symlink_deps
+
+%clean
+rm -rf %{buildroot} %{npm_cache_dir}
+
+%if 0%{?enable_tests}
+%check
+%{nodejs_symlink_deps} --check
+#$CHECK
```
Comment (ehelms) [inline nodejs-react-onclickoutside/nodejs-react-onclickoutside.spec]: This also seems like abug in npm2rpm

**Comment by ekohl** [comment]
Updated

File: nodejs-react-onclickoutside/nodejs-react-onclickoutside.spec [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+License: MIT
+URL: https://github.com/Pomax/react-onclickoutside
+Source0: https://registry.npmjs.org/%{npm_name}/-/%{npm_name}-%{version}.tgz
+BuildRequires: nodejs-packaging
+BuildArch:  noarch
+ExclusiveArch: %{nodejs_arches} noarch
+
+%{?nodejs_find_provides_and_requires}
+
+%define npm_cache_dir /tmp/npm_cache_%{name}-%{version}-%{release}
```
Comment (ehelms) [inline nodejs-react-onclickoutside/nodejs-react-onclickoutside.spec]: This appears to be unused for non-bundled NPMs, so for cleanness sake's we should remove references to it (and update npm2rpm)

File: nodejs-react-onclickoutside/nodejs-react-onclickoutside.spec [hunk 4]
```diff
@@ -0,0 +1,46 @@
+%global npm_name react-onclickoutside
+%global enable_tests 0
+
+%{?nodejs_find_provides_and_requires}
```
Comment (ekohl) [inline nodejs-react-onclickoutside/nodejs-react-onclickoutside.spec]: I just noticed this is duplicated as well

**Comment by ehelms** [comment]
@ekohl Were you planning another test round of changes to npm2rpm for this? Or should we take it as is and build?

**Comment by ekohl** [comment]
@ehelms I have a huge list of NPM packages that need updates (including react itself). This needs updates. Shall I make this a large PR with 1 package per commit?

**Comment by ehelms** [comment]
I'm fine with that. This one is needed to unblock nightlies only? Or the other updates as well?

**Comment by ekohl** [comment]
Since it includes react and others I honestly don't know what breaks next.

**Comment by ehelms** [comment]
Feel free to add the rest here then.

**Comment by ekohl** [comment]
@dLobatog that works too. Today I'm going to work on all the needed package updates.

**Comment by dLobatog** [comment]
nightly - http://ci.theforeman.org/view/Packaging/job/packaging_build_rpm/9617/

**Pull request merged**